src/main/kotlin/org/codefreak/codefreak/util/FileUtil.kt

```diff
@@ -0,0 +1,32 @@
+package org.codefreak.codefreak.util
+
+import java.nio.file.attribute.PosixFilePermission
+
+object FileUtil {
+  private val stripPrefixPattern = """^(?:\.*/)*(?:\.+$)?""".toRegex()
+
+  /**
+   * Remove leading dots and slashes from given path
+   */
+  fun normalizeName(name: String) = name.trim().trim('/').replace(stripPrefixPattern, "").trim()
```

Comment:
We should be using something like `sanitizeName` as name because this is actually a security measure.
Maybe we can use something like `Paths.get("/").resolve("foo/../../../etc/passwd").normalize()` without relying on a RegEx. This will also resolve weird intermediate paths.